addOns/ascanrulesAlpha/src/main/java/org/zaproxy/zap/extension/ascanrulesAlpha/LdapInjectionScanRule.java

```diff
@@ -664,20 +634,18 @@ private boolean checkResultsForLDAPAlert(
                         .setMessage(attackMessage)
                         .raise();
 
-                if (log.isDebugEnabled()) {
-                    String logMessage =
-                            MessageFormat.format(
-                                    "A likely LDAP injection vulnerability has been found with [{0}] URL [{1}] "
-                                            + "on parameter [{2}], using an attack with LDAP meta-characters [{3}], "
-                                            + "yielding known [{4}] error message [{5}], which was not present in the original response.",
-                                    getBaseMsg().getRequestHeader().getMethod(),
-                                    getBaseMsg().getRequestHeader().getURI().getURI(),
-                                    parameterName,
-                                    errorAttack,
-                                    LDAP_ERRORS.get(errorPattern),
-                                    errorPattern);
-                    log.debug(logMessage);
-                }
+                String logMessage =
+                        MessageFormat.format(
```

Comment:
This should be replaced with the debug call (or leave the `isDebugEnabled` check), to avoid creating the string.